test/TestProjects/MgmtDiscriminator/Generated/Models/DeliveryRuleAction.cs

```diff
@@ -12,10 +12,10 @@ namespace MgmtDiscriminator.Models
     /// Please note <see cref="DeliveryRuleAction"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
     /// The available derived classes include <see cref="DeliveryRuleCacheExpirationAction"/>, <see cref="DeliveryRuleCacheKeyQueryStringAction"/>, <see cref="DeliveryRuleRequestHeaderAction"/>, <see cref="DeliveryRuleResponseHeaderAction"/>, <see cref="OriginGroupOverrideAction"/>, <see cref="DeliveryRuleRouteConfigurationOverrideAction"/>, <see cref="UrlRedirectAction"/>, <see cref="UrlRewriteAction"/> and <see cref="UrlSigningAction"/>.
     /// </summary>
-    public partial class DeliveryRuleAction
+    public abstract partial class DeliveryRuleAction
```

Comment:
Not enable `EnableAbstractBaseClassWithDiscriminator` but why this is changed to abstract?

Reply:
Has changed the configuration to a list `suppress-abstract-base-class`